docs/extend/extend-apiml/api-mediation-oidc-authentication.md

```diff
@@ -26,20 +27,28 @@ The API ML Gateway can then create mainframe user credentials (JWT or a Passtick
 
 ## Authentication Flow
+
 The following diagram illustrates the interactions between the participants of the OIDC/OAuth2 based API ML authentication process.
 
 ![APIML OIDC Workflow](../../images/api-mediation/apiml-oidc-auth-seq.png)
 
-* When a user wants to access mainframe resources or services using the client application without a valid authentication / access token, the client redirects the user agent to the login end-point of the distributed OIDC provider. 
-* The user is asked to provide valid credentials (authentication factors).
-* After successful validation of all authentication factors, the OIDC provider grants the client an Access Token.
-* The client can then request from API ML Gateway the needed mainframe resources presenting the access token in the request. 
-* The Gateway validates the access token at the provider's OIDC/introspection end-point. If the access token is validated, the outcome is cached for a short time (20 sec by default). 
-* In subsequent calls with the same token, the Gateway reuses the cached validation outcome. As such, round trips to the OIDC /introspection end-point are not required between short intervals, when the client needs to access multiple resources in a row to complete a unit of work. The caching interval is configurable with a default value of 20 seconds, which is typically a sufficient time to allow most client operations requiring multiple API requests to complete, while also providing adequate protection against unauthorized access. 
-* The API ML Gateway fetches the distributed user identity from the distributed access token and maps this user identity to the user mainframe identity using SAF. 
-* The API ML Gateway calls the requested mainframe service/s with mainframe user credentials (Zowe, SAF JWT, or pass-ticket) which are expected by the target mainframe service.
+- When a user wants to access mainframe resources or services using the client application without a valid authentication or an access token, the client redirects the user agent to the login end-point of the distributed OIDC provider.
+- The user is asked to provide valid credentials (authentication factors).
+- After successful validation of all authentication factors, the OIDC provider grants the client an Access Token.
+- The client can then request from API ML Gateway the needed mainframe resources presenting the access token in the request.
+
+- The Gateway validates the access token.
+
+- The Gateway validates the access token by comparing the key id of the token against the key ids obtained from the authorization server's JWK keys endpoint,URL to end point should be set using the property jwks_uri. If the access token is validated, the outcome is cached for a short time (20 sec by default).
+
+- The JWK Keys obtained from the authorization server's endpoint are cached for a while to prevent the repeated calls to the endpoint. It can be set using the property jwks.refreshInternalHours (it's 1 hour by default)
```

Comment:
Please make "for a while" more specific. Perhaps, " The JWK Keys obtained from the authorization server's endpoint can be cached for a specified interval to prevent the repeated calls to the endpoint."